package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stream-io/stream-chat-css",
-  "version": "4.17.4",
+  "version": "5.0.0-rc.6",
```

Comment:
Please adjust the release configuration. Releases cut from `main` are still published as `rc`.

Reply:
hm, I think the config is fine, the version is only `rc` here because we're on the `rc` branch. But once we merge this to `main` the release plugin should update the version to v5 (without the rc postfix)

Reply:
This is the config for reference: https://github.com/GetStream/stream-chat-css/blob/rc/package.json#L28

Reply:
Next version is computed based on git tags, not what's in `package.json`, maybe that's what confused you @oliverlaz?

Reply:
@szuperaz  no no, I just didn't realize we have a default config for `main`, and a custom one for the `rc` branch:
```
    "branches": [
      "main",
      {
        "name": "rc",
        "prerelease": true
      }
    ],
```

I interpreted this block wrong :)